lua/nvim-tree/actions/node/buffer.lua

```diff
@@ -0,0 +1,61 @@
+-- Copyright 2019 Yazdani Kiyan under MIT License
+local notify = require("nvim-tree.notify")
+
+local M = {}
+
+---@param node Node
+---@param opts ApiNodeDeleteWipeBufferOpts|nil
+---@return nil
+function M.delete(node, opts)
+  M.delete_buffer("delete", node.absolute_path, opts)
```

Comment:
This handles nil or partial node gracefully, however not a malformed node like `{ aaaabsolute_path = "foo" }`

That's OK - garbage in, garbage out.